lib/python/pyflyby/_flags.py

```diff
@@ -4,6 +4,7 @@
 
 from __future__ import absolute_import, division, with_statement
 
+from __future__ import print_function
```

Comment:
This should be combined with the above import.  I don't recall if it works to have multiple time machine statements because it is handled in a special way.